src/chronometer.js

```diff
@@ -1,33 +1,48 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
-  }
-
+  this.intervalId = setInterval (() => {
+  this.currentTime++
+  if (printTimeCallback) return printTimeCallback()
+  }, 1000);
+}
   getMinutes() {
-    // ... your code goes here
+    if (!this.currentTime) return 0
+else {
+return Math.floor(this.currentTime / 60)
   }
+}
 
   getSeconds() {
-    // ... your code goes here
+    if (!this.currentTime) return 0
```

Comment:
Same as before, this if statement is unnecessary!
`return Math.floor(this.currentTime % 60)` ✅